packages/compiler/lib/lib.tsp

```diff
@@ -120,6 +120,12 @@ scalar duration;
  */
 scalar boolean;
 
+/**
+ * Represent a 32bit unix timestamp datetime.
```

Comment:
I think it's important to mention that this has 1 second granularity, so should not be used to represent e.g. timestamps with millisecond granularity (which now that I think about it is super common and we should probably have a datatype for it).